crates/winscard/src/lib.rs

```diff
@@ -11,13 +11,15 @@ mod macros;
 mod ber_tlv;
 mod card_capability_container;
 mod chuid;
+#[cfg(feature = "scard")]
 mod compression;
 mod dummy_rng;
 /// Contains env variables names that represent smart card credentials.
 #[cfg(feature = "std")]
 pub mod env;
 mod piv_cert;
 mod scard;
+#[cfg(feature = "scard")]
 mod scard_context;
```

Comment:
Why did you make the `scard_context` optional? I mean that the `winscard` crate would be useless if the user will not be able to acquire the scard context.